SimpleWAWS/Code/CsmExtensions/CsmSubscriptionExtensions.cs

```diff
@@ -84,6 +145,31 @@ public static MakeSubscriptionFreeTrialResult MakeTrialSubscription(this Subscri
 
             return result;
         }
+        public static MakeSubscriptionFreeTrialResult MakeJenkinsTrialSubscription(this Subscription subscription)
```

Comment:
It will need to be a lot different , I was working around for creating multiple resourcegroups in a region instead of just one, by specifying the region names multiple times in the environment variables. For now I'll merge them back into one